charts/kube-state-metrics/templates/deployment.yaml

```diff
@@ -96,12 +96,17 @@ spec:
         {{- if .Values.kubeconfig.enabled }}
         - --kubeconfig=/opt/k8s/.kube/config
         {{- end }}
+        {{- if .Values.kubeRBACProxy.enabled }}
+        - --telemetry-host=127.0.0.1
+        - --telemetry-port={{ $telemetryPort }}
+        {{- else }}
         {{- if .Values.selfMonitor.telemetryHost }}
         - --telemetry-host={{ .Values.selfMonitor.telemetryHost }}
         {{- end }}
         {{- if .Values.selfMonitor.telemetryPort }}
         - --telemetry-port={{ .Values.selfMonitor.telemetryPort | default 8081 }}
```

Comment:
```suggestion
        - --telemetry-port={{ $telemetryPort }}
```

Reply:
I thought about doing this. As this is the "no rbac proxy" path, this will always take the value from the `values.yaml` file, so decided against changing the initial implementation, both will produce the same value. I can apply this change for consistency perhaps, either is fine imo

Reply:
@mrueg, do you agree to merge this as-is, or should we ask @Thiryn to make the change?

Reply:
I just would want to avoid changing the "default 8081" at some point and missing this one here.